thinkhazard_processing/processing.py

```diff
@@ -165,39 +159,36 @@ def create_outputs(hazardset, layers, readers):
         else:
             bbox = bbox.intersection(polygon)
 
+    simplified = func.ST_Simplify(AdministrativeDivision.geom, 0.005) \
+        .label('simplified')
     admindivs = DBSession.query(AdministrativeDivision) \
+        .add_columns(simplified) \
         .filter(AdministrativeDivision.leveltype_id == adminlevel_REG.id)
 
     if hazardset.local:
-        # TODO : this could be optimized (double reprojection)
-        # Better to had a geom_4326 column
         admindivs = admindivs \
-            .filter(
-                func.ST_Transform(AdministrativeDivision.geom, 4326)
-                .intersects(
+            .filter(AdministrativeDivision.geom.intersects(
                     func.ST_GeomFromText(bbox.wkt, 4326))) \
-            .filter(func.ST_Intersects(
-                func.ST_Transform(AdministrativeDivision.geom, 4326),
-                func.ST_GeomFromText(bbox.wkt, 4326)))
+            .filter(AdministrativeDivision.geom.intersects(
```

Comment:
The original query was "intersects" (spatial index on bbox) and "ST_Intersects" (strict geometric predicate).
It seems you replace this by "intersects" and "intersects".